extensions/sql-migration/src/wizard/assessmentResultsDialog.ts

```diff
@@ -0,0 +1,133 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as mssql from '../../../mssql';
+import { MigrationStateModel } from '../models/stateMachine';
+
+export class AssessmentResultsDialog {
+
+	private static readonly OkButtonText: string = 'OK';
+	private static readonly CancelButtonText: string = 'Cancel';
+
+	// protected _onSuccess: vscode.EventEmitter<T> = new vscode.EventEmitter<T>();
+	protected _isOpen: boolean = false;
+	// public readonly onSuccess: vscode.Event<T> = this._onSuccess.event;
+	public dialog: azdata.window.Dialog | undefined;
+
+	private assessmentTable: azdata.TableComponent | undefined;
+
+	// Dialog Name for Telemetry
+	public dialogName: string | undefined;
+
+	constructor(public ownerUri: string, public model: MigrationStateModel, public title: string) {
+	}
+
+	protected async updateModel(): Promise<void> {
+		return undefined;
+	}
+
+	protected async initializeDialog(dialog: azdata.window.Dialog): Promise<void> {
+		dialog.registerContent(async view => {
+			this.assessmentTable = view.modelBuilder.table()
+				.withProperties({
+					columns: [
+						'Target',
```

Comment:
I'm guessing we'll localize these later?

Reply:
these are just placeholders.  the actual UX design won't have these exact strings, so no need to add localized constants now.